code/modules/vore/fluffstuff/custom_permits_vr.dm

```diff
@@ -23,16 +23,6 @@
 	This license expires on DD/Mon/YYYY and must be renewed by CentCom prior to this date."}
 */
 
-// bwoincognito:Tasald Corlethian
-/obj/item/fluff/permit/tasald_corlethian
```

Comment:
This doesn't need to be deleted. Why is it being deleted? It's not even expired yet.